src/components/SingleArticle.jsx

```diff
@@ -30,6 +31,9 @@ if (isLoading) {
         <p>{article.body}</p>
         <img src={article.article_img_url} />
         <p>Author: {article.author}</p>
```

Comment:
is there any other information about the article that the user might want to know? 
see this on a more interesting way to present info: https://www.refactoringui.com/previews/labels-are-a-last-resort